catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/QueryResponseImpl.java

```diff
@@ -50,6 +50,8 @@
 
     List<Result> resultList = null;
 
+    private long timeout;
```

Comment:
It can be useful to include the units in the variable name like `timeoutMillis`.